expyfun/_experiment_controller.py

```diff
@@ -940,6 +944,8 @@ def get_presses(self, live_keys=None, timestamp=True, relative_to=None):
             If timestamp==False, returns a list of strings indicating which
             keys were pressed. Otherwise, returns a list of tuples
             (str, float) of keys and their timestamps.
+            If releases is True, the last value in the tuple will be a string,
+            either 'press' or 'release'.
```

Comment:
The reason I went for this initially, is that it won't break existing code that doesn't use releases.

If people have a 
```python
for key, time in presses:
    doSomething()
```
It would break if you added a third element. But if we don't mind then we could return a 3-tuple or a dict or even a `ResponseEvent` object.